src/Device.ts

```diff
@@ -188,7 +190,6 @@ export function detectDevice(): BuiltinHandlerName | undefined
 		else if (
 			engineName === 'webkit' &&
 			isIOS &&
-			osVersion >= 14.3 &&
```

Comment:
I understand this is not super required, but what is wrong with it? Is this version requirement generating some issue?

Reply:
Yes, iPadOS WKWebView reports as OS Version 10.15.